### PR TITLE
Works with Github actions

### PR DESCRIPTION
**Pull request by Jeckerson**
* Implement tests for PostgreSQL
* Separate tests them (if needed)
* Implement actions to run all tests

### CODE REVIEW

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+        image: postgres:10.8
+        ports:
+          - 5432/tcp
+        options: --health-cmd pg_isready --health-interval 10s --health-timeout 5s --health-retries 3
+    strategy:
+      fail-fast: false
+      matrix:
+        php-versions: ['7.2', '7.3', '7.4']
+    steps:
+      - uses: actions/checkout@v1
```
Comment (ruudboon) [inline .github/workflows/tests.yml]: Can you name every step?

File: .github/workflows/tests.yml [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+      - name: Run test suite
+        env:
+          MYSQL_TEST_DB_PORT: ${{ job.services.mysql.ports['3306'] }}
+          POSTGRES_TEST_DB_PORT: ${{ job.services.postgres.ports['5432'] }}
+        if: always()
+        run: vendor/bin/phpunit --coverage-clover=coverage.xml
+
+      - name: Upload coverage to Codecov
+        if: always()
```
Comment (ruudboon) [inline .github/workflows/tests.yml]: Does it make sense to upload after step fail?

File: .github/workflows/tests.yml [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+          restore-keys: ${{ runner.os }}-composer-
+
+      - name: Install Composer dependencies
+        run: composer install --prefer-dist --no-suggest
+
+      - name: Run test suite
+        env:
+          MYSQL_TEST_DB_PORT: ${{ job.services.mysql.ports['3306'] }}
+          POSTGRES_TEST_DB_PORT: ${{ job.services.postgres.ports['5432'] }}
+        if: always()
```
Comment (ruudboon) [inline .github/workflows/tests.yml]: Not sure if you wanna run if composer fails

File: .github/workflows/validations.yml [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+        run: ~/.composer/vendor/bin/phpcs
+
+  validate-code-static:
+    name: Static Code with PHP ${{ matrix.php-versions }}
+    runs-on: ubuntu-latest
+    needs: validate-code-style
+    strategy:
+      fail-fast: false
+      matrix:
+        php-versions: ['7.2', '7.3', '7.4']
```
Comment (ruudboon) [inline .github/workflows/validations.yml]: Maybe running the validation the latest is good enough?

File: .github/workflows/tests.yml (hunk 1 above)
Comment (Jeckerson) [reply .github/workflows/tests.yml]: 7928745

File: .github/workflows/validations.yml (hunk 4 above)
Comment (Jeckerson) [reply .github/workflows/validations.yml]: To make sure that code is valid for every version. Example: deprecation warnings.

File: .github/workflows/tests.yml (hunk 2 above)
Comment (Jeckerson) [reply .github/workflows/tests.yml]: afa39a8

File: .github/workflows/tests.yml (hunk 3 above)
Comment (Jeckerson) [reply .github/workflows/tests.yml]: afa39a8

**Pull request merged**